src/utils/__mocks__/next-cache.ts

```
@@ -0,0 +1 @@
+export const unstable_cache = (fn: Function) => fn
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)